x-pack/plugin/sql/sql-action/src/main/java/org/elasticsearch/xpack/sql/action/AbstractSqlQueryRequest.java

```diff
@@ -6,7 +6,6 @@
  */
 package org.elasticsearch.xpack.sql.action;
 
-import org.elasticsearch.TransportVersions;
```

Comment:
This looks weird in this PR, but I guess if it compiles it is not an issue to include it.